valhalla/userrequests/test/test.py

```diff
@@ -127,17 +128,37 @@ def setUp(self):
             exposure_time=60, exposure_count=1, type='LAMPFLAT'
         )
 
+        self.sbig_fixed_overhead_per_exposure = 1
+        self.sbig_filter_change_time = 2
+        self.sbig_front_padding = 90
+        self.sbig_readout_time2 = 14.5
+
+        self.floyds_fixed_overhead_per_exposure = 0.5
+        self.floyds_filter_change_time = 0
+        self.floyds_front_padding = 240
+        self.floyds_readout_time1 = 25
```

Comment:
Similarly, there only appears to be one floyds readout time, not sure why there is a 1.